app/controllers/documents_controller.rb

```diff
@@ -36,50 +35,34 @@ def archive
   end
 
   def new
-    @document_form = DocumentWizard::UploadForm.new
+    @document = @planning_application.documents.build
   end
 
   def create
-    @document_form = DocumentWizard::ConfirmUploadForm.new(
-      document_upload_confirmation_params,
-    )
+    @document = @planning_application.documents.build
+    unless document_params[:file].nil?
```

Comment:
Rather than checking to see if the file is present in the controller, could we not add presence validation for file in the document model? I don't think there is any situation we would want a document without a file attached? And that would simplify this controller action even more!